packages/design-system/src/components/card/MyBookmarkCard.tsx

```diff
@@ -0,0 +1,58 @@
+import { Icon } from '@/icons';
+import chippiNoImage from '../../assets/chippi_no_image.svg';
+import BaseCard from './BaseCard';
+
+interface MyBookmarkCardProps {
+  title: string;
+  content?: string;
+  category: string;
+  imageUrl?: string;
+  date?: string;
+}
```

Comment:
```suggestion
}type BaseProps = {
  title: string;
  content?: string;
  category: string;
  imageUrl?: string;
};

type RemindProps = BaseProps & {
  type: 'remind';
  timeRemaining: string; // 필수
  date?: never;         // 금지
};

type BookmarkProps = BaseProps & {
  type: 'bookmark';
  date: string;         // 필수
  timeRemaining?: never; // 금지
};

type CardProps = RemindProps | BookmarkProps;
```
타입 분리해서 사용하는거 좋을거같아요-!! 다만 이런식으로 타입별 필수,불가 필드를 분리해서 사용하는 방식은 어떤가요?

Reply:
헉. 제가 두 개 합친 `Card`로 두면서 `interface`가 참 애매하다고 생각했는데, `type` 키워드로 `확장`하는 방법이 있었군요!
반영했습니다~~ 👍

Reply:
[dc15864](https://github.com/Pinback-Team/pinback-client/pull/40/commits/dc158646c3f707a10429145042c3aaf9b07cb720)